pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -325,7 +325,8 @@ func (rq *ResourceQuotaController) syncResourceQuota(v1ResourceQuota *v1.Resourc
 			Namespace:       resourceQuota.Namespace,
 			ResourceVersion: resourceQuota.ResourceVersion,
 			Labels:          resourceQuota.Labels,
-			Annotations:     resourceQuota.Annotations},
+			Annotations:     resourceQuota.Annotations,
```

Comment:
@derekmahar and @deads2k, both solutions are ok to me, do we have plan to merge #45552 right now?